pack/valk_encounter.json

```diff
@@ -34,13 +37,15 @@
 		"boost": 2,
 		"code": "25030",
 		"faction_code": "encounter",
+		"flavor": "Behind Amora's charm lies a cruel ambition and the power to subdue those in her way.",
```

Comment:
```suggestion
		"flavor": "Behind Amora's charm lies cruel ambition and the power to subdue those in her way.",
```

Reply:
This suggestion is based on the official text on the card.

<img width="917" alt="Screenshot 2022-01-22 at 12 06 49" src="https://user-images.githubusercontent.com/67125/150637911-b825e366-f0ff-4d00-b2ef-8f54b7f012ec.png">